### DISCUSSION TITLE
Request for Assistance: Issue with Missing Script.sol, Test.sol, and Vm.sol Files

### DISCUSSION THREAD

**Discussion by chapman91** [Cyfrin/foundry-full-course-cu#2215] [Q&A] [answered]
**_Hey!_**

I'm currently working on a project using Foundry, and I've hit a roadblock that I could use some help with. I'm encountering an issue where the Script.sol, Test.sol, and Vm.sol files from the forge-std library seem to be missing from my project. Here's a quick rundown of the situation and what I've tried so far:

**The Problem:**
I'm receiving errors indicating that the files Script.sol, Test.sol, and Vm.sol are missing when I attempt to run my tests.

**Steps I've Tried:**
Verified the File Path:

- I navigated to the lib/forge-std/src/ directory where these files should be located. Unfortunately, they are not present.
 Checked for remappings.txt:

-I reviewed my remappings.txt file to ensure that the forge-std library is correctly mapped. However, I’m unsure if the remapping is pointing to the right path or if there's another issue at play.
Suspected Cause:

-I believe the missing files might be causing a test to fail. The test is supposed to verify the price feed version, but it’s throwing a revert error, possibly because the correct mock contracts or dependencies aren’t being deployed due to the missing files.

Before I try reinstalling forge-std or making any changes, I wanted to reach out to the community for guidance. Has anyone encountered a similar issue, or do you have any suggestions on how I can resolve this?

**Comment by EngrPips** [comment, marked as answer]
Hello @chapman91, You should run `forge install foundry-rs/forge-std` and let's see if the issue still persists.

**Reply by chapman91** [reply]
I had to delete the folder and reinstall it. Thanks! I felt very nervous doing it not knowing the consequences.

**Reply by EngrPips** [reply]
I am glad you took the courage to do it, hahaha. Keep crushing it.
